src/node/config.ts

```diff
@@ -232,6 +232,28 @@ export interface BuildConfig extends SharedConfig {
    * @internal
    */
   ssr?: boolean
+  /**
+   * Set to `false` will produce modular javascript code,
+   * which requires browser's native ES module support,
+   * other options will use SystemJS to load each module instead.
+   * Set to `true` and `'inline'` will insert the SystemJS runtime to a inline script tag,
+   * set to `{ cdn: '...' }` will load the SystemJS runtime via a CDN you specified.
+   *
+   * Example `vite.config.js`:
+   * ```js
+   * module.exports = {
+   *   // will insert a inline script tag contains SystemJS runtime
+   *   useSystemJs: true,
+   *   useSystemJs: 'inline',
+   *
+   *   // will load SystemJS runtime via CDN
+   *   useSystemJs: { cdn: '//unpkg.com/systemjs@6/dist/s.min.js' }
+   * }
+   * ```
+   *
+   * @default false
+   */
+  useSystemJs?: boolean | 'inline' | { cdn: string }
```

Comment:
I feel this can be simplified to `boolean | string` where `true` implies inline, and string value will be used as a CDN URL.